src/Compilers/CSharp/Portable/Symbols/TypeWithAnnotations.cs

```diff
@@ -244,6 +252,7 @@ internal TypeWithAnnotations MergeEquivalentTypes(TypeWithAnnotations other, Var
 
         public bool IsResolved => _extensions?.IsResolved != false;
         public TypeSymbol Type => _extensions?.GetResolvedType(DefaultType);
+        public NullableAnnotation NullableAnnotation => _extensions?.GetResolvedAnnotation(DefaultNullableAnnotation) ?? default;
```

Comment:
If we used `-1` for the "uninitialized" value of `DefaultNullableAnnotation`, we could assert that that value is never returned here.
I'm not sure we can do that with `Ignored`.